build.gradle

```diff
@@ -3,6 +3,7 @@ apply plugin: 'checkstyle'
 
 repositories {
     mavenCentral()
+    
```

Comment:
this empty line has no purpose, please remove it

Reply:
Empty line has been removed